user.js

```diff
@@ -1279,14 +1279,14 @@ user_pref("mail.identity.default.compose_html", false);
  * 2=HTML (only send a HTML part)
  * 3=both (send both the HTML part and the plain text alternative part)
  * [1] https://drewdevault.com/2016/04/11/Please-use-text-plain-for-emails.html ***/
-user_pref("mail.default_send_format", 1);
+user_pref("mail.default_send_format", 0);
```

Comment:
```suggestion
    // user_pref("mail.default_send_format", 0);
```

Let's disable it if we revert to default.